### DISCUSSION TITLE
web3.exceptions.InvalidAddress: ('Address has an invalid EIP-55 checksum. After looking up the address from the original source, try again

### DISCUSSION THREAD

**Discussion by ptn77** [smartcontractkit/full-blockchain-solidity-course-py#968] [Q&A] [answered]
When getting the latest transaction, I'm using the my_address variable with the ganache address set to it and I'm getting this error:
web3.exceptions.InvalidAddress: ('Address has an invalid EIP-55 checksum. After looking up the address from the original source, try again

I'm not seeing the nonce = 0 as the course shows. What am I doing wrong?

**Comment by ptn77** [comment, marked as answer]
I re-copied the address from ganache and it's working now. Sorry

**Reply by cromewar** [reply]
Nice catch, I'm glad you solved it so fast by yourself.
Feel free to make any other question.
